File: ePrijevozSarajevo/UI/eprijevoz_desktop/lib/models/type.dart
import 'package:json_annotation/json_annotation.dart';

part 'type.g.dart';

@JsonSerializable()
class Type {
  int? typeId;
  String? name;
  DateTime? modifiedDate;

  Type({this.typeId, this.name});

  factory Type.fromJson(Map<String, dynamic> json) => _$TypeFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$TypeToJson(this);
}

File: ePrijevozSarajevo/ePrijevozSarajevo.Services/Database/Type.cs
namespace ePrijevozSarajevo.Services.Database
{
public class Type
    {
        public int TypeId { get; set; }
        public string Name { get; set; } = null!;
        public DateTime? ModifiedDate { get; set; }
        public int? CurrentUserId { get; set; }
    }
}
